umpf

```diff
@@ -263,6 +268,20 @@ setup() {
 			GIT_REMOTE="${1}"
 			shift
 			;;
+		--override)
+			IFS=: read -r topic commitish <<< "${1}"
+			if [ -z "${commitish}" ]; then
+				commitish="${topic}"
+			fi
+
+			rev="$(${GIT} rev-parse "${commitish}^{}" 2> /dev/null)"
+			if [ -z "${rev}" ]; then
+				echo "Revision ${commitish} not found for topic ${topic}"
+				exit 1
+			fi
+			OVERRIDES[$topic]="${rev}"
```

Comment:
This is lost for `umpf continue`. It need to be stored in `bailout()` in some way and recovered in `do_continue()`

Reply:
Fixed.